pythonrc.py

```diff
@@ -109,6 +108,25 @@
 )
 
 
+if sys.version_info < (3, 7):
+    import imp
+    def find_module(name):
+        """Search for a module"""
+        (_, pkg_path, _) = imp.find_module(name)
+        return pkg_path
+else:
+    def find_module(name):
+        """Search for a module"""
+        spec = importlib.util.find_spec('name')
```

Comment:
Thanks for your patchset. I appreciate your help in ensuring this works for the latest version of python (I don't often use the latest versions). That said, on a cursory glace (I don't know the importlib.utils api), shouldn't this ought to be ?
```
            spec = importlib.util.find_spec(name)
```

ie: instead of the string 'name' ?

Reply:
Yes, thanks for finding that error.